aspnetcore/fundamentals/configuration/index.md

```diff
@@ -757,6 +768,8 @@ public class Program
 }
 ```
 
+The base path is set with <xref:Microsoft.Extensions.Configuration.FileConfigurationExtensions.SetBasePath*>. `SetBasePath` is made available to an app by referencing the [Microsoft.Extensions.Configuration.FileExtensions](https://www.nuget.org/packages/Microsoft.Extensions.Configuration.FileExtensions/) package, which is made available to any app that references the [Microsoft.AspNetCore.App metapackage](xref:fundamentals/metapackage-app).
```

Comment:
Do we really need to tell them it's in M.A.App? Guess it doesn't hurt.

Reply:
Yeah, the reader could land into any of these sections from an internal or external bookmark. If we only say the package, they might add the package to an app that already references `App`.